controller/node_controller.go

```diff
@@ -1089,6 +1108,72 @@ func (nc *NodeController) syncNFSClientVersion(kubeNode *corev1.Node, node *long
 		fmt.Sprintf("NFS clients %v not found. At least one should be enabled", nfsClientVersions))
 }
 
+func checkModulesLoadedUsingkmod(checkingModules map[string]string) (map[string]string, error) {
+	kmodResult, err := lhexec.NewExecutor().Execute(nil, "kmod", []string{"list"}, lhtypes.ExecuteDefaultTimeout)
+	if err != nil {
+		return nil, err
+	}
+
+	notFoundModulesUsingkmod := map[string]string{}
```

Comment:
nit: make the names of variables simple (not lengthy) and readable

```suggestion
	notFoundModules:= map[string]string{}
```

Reply:
Modified. Thanks